splitio/client/client.py

```diff
@@ -116,12 +117,13 @@ def _make_evaluation(self, key, feature, attributes, method_name, metric_name):
                 bucketing_key,
                 utctime_ms(),
             )
-
             self._record_stats([(impression, attributes)], start, metric_name)
```

Comment:
I would encourage to use the record_stats method for tracking latency times on method. you have there you long it takes

Reply:
I don't have method name though in record_stats, I would need to pass it